rbac/server/api/utils.py

```diff
@@ -39,6 +42,29 @@ def validate_fields(required_fields, body):
         raise ApiBadRequest("Bad Request: Improper JSON format")
 
 
+def create_authorization_response(token, data):
+    response = json({"data": data})
+    response.cookies[SIGNATURE_KEY] = ".".join(token.split(".")[0:2])
+    response.cookies[PAYLOAD_KEY] = token.split(".")[2]
+
+    response.cookies[SIGNATURE_KEY]["httponly"] = True
+    return response
+
+
+def extract_request_token(request):
+    try:
+        return request.json["tracker"]["slots"]["token"]
+    except (KeyError, TypeError):
+        pass
+    if request.cookies.get(SIGNATURE_KEY) is None:
```

Comment:
Are you trying to check if it is None or an empty string here?

Reply:
None.